contact.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Home</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+    <div class="container contact">
+        <header class="top-nav">
+            <img src="img\lambda-black.png" alt="Logo">
+            <nav>
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
+
+        <section class="contact-us">
+
+            <div class="contact-us-header">
+                <h1>Contact Us</h1>
+                <h3>We would love to help with any difficulties</h3>
+            </div>
+
+            <div class="contact-form">
+                <form action="/action_page.php">
+                <p>First name:</p>
```

Comment:
you will want to use the <label> element for forms in the future, it's worth reading the semantics for those on mdn when you have the chance